ssas/groups_test.go

```diff
@@ -71,12 +71,35 @@ func (s *GroupsTestSuite) TestCreateGroup() {
 	assert.Nil(s.T(), err)
 	g, err := CreateGroup(gd)
 	assert.Nil(s.T(), err)
-	_ = CleanDatabase(g)
+	err = CleanDatabase(g)
+	assert.Nil(s.T(), err)
 	gd.ID = ""
 	_, err = CreateGroup(gd)
 	assert.EqualError(s.T(), err, "group_id cannot be blank")
 }
 
+func (s *GroupsTestSuite) TestUpdateGroup() {
+	groupBytes := []byte(SampleGroup)
+	gd := GroupData{}
+	err := json.Unmarshal(groupBytes, &gd)
+	assert.Nil(s.T(), err)
+	g, err := CreateGroup(gd)
```

Comment:
Would it be better to create the group directly? This test relies on `CreateGroup()`, which isn't being tested here.